### PR TITLE
Fix(UI)#9017: Fixed description not rendering on explore page in Safari browser

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on #9017 

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
<p align="center">

Before: 

Screenshot 2022-12-05 at 4 16 11 PM

After:

Screenshot 2022-12-05 at 4 18 24 PM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.less [hunk 1]
```diff
@@ -71,7 +71,7 @@
     padding: 0px;
     margin: 0px;
     line-height: 0px;
-    height: auto;
+    height: 16px;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.less:74]: @aniketkatkar97, Can you explain why we are adding fixed height?

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less [hunk 2]
```diff
@@ -273,3 +273,10 @@
   flex-direction: column;
   padding: 24px 16px 0px 16px;
 }
+
+.description-text {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  line-height: 18px;
+}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/app.less:282]: @aniketkatkar97 , Why did we remove these properties?

```css
   max-height: 64px;
  -webkit-line-clamp: 3;
  -webkit-box-orient: vertical;
```

File: openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.less (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.less:74]: Height auto is working fine for 'Chrome' but it is giving height 0 in the 'Safari' browser. As the button is of type link. I have given it the height same as the default font size.

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/styles/app.less:282]: This was older styling given to limit 3 lines for description. As we are handling that logically in the component. we don't need that styling. Also, this styling is what was causing the original issue in 'Safari' in the first place

**Review by ShaileshParmar11 — COMMENTED** [review]
Existing implementation
image

with your changes:
image

We need to re-think about the changes

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less (hunk 2 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/styles/app.less:282]: Okay, Can we make sure to show 3 lines in the explore page card as we had before?

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/styles/app.less:282]: Sure, looking into that

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**